.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,27 @@
+name: Dependency Submission
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
To prevent the pull request checks from failing, I think we need to disable the dependency submission for pull_requests here. The issue is that pull request creators don't have enough permissions to do the dependency submission step.

Long story short - could you remove lines 6 & 7 to disable this check for PRs?

Reply:
Done